backend/official_plugins/tracim_backend_child_removal/__init__.py

```diff
@@ -16,14 +16,14 @@ class ChildRemovalPlugin:
     """
 
     @hookimpl
-    def on_user_role_in_workspace_deleted(
-        self, role: UserRoleInWorkspace, context: TracimContext
+    def on_user_config_in_workspace_deleted(
+        self, config: UserConfigInWorkspace, context: TracimContext
```

Comment:
"config" is too ambiguous, the concept is user_workspace_config so it's better to use its full name

Reply:
yes it has been changed

Reply:
done